images/runtime/download-cni.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copyright 2017-2020 Authors of Cilium
+# SPDX-License-Identifier: Apache-2.0
+
+set -o xtrace
+set -o errexit
+set -o pipefail
+set -o nounset
+
+cni_version="0.7.5"
+
+for arch in amd64 arm64 ; do
+  curl --fail --show-error --silent --location "https://github.com/containernetworking/plugins/releases/download/v${cni_version}/cni-plugins-${arch}-v${cni_version}.tgz" --output "/tmp/cni-${arch}.tgz"
```

Comment:
Thanks, I've added these, but instead of downloading the checksum at runtime I add a file where checksums are declared, since the checksum file itself can be compromised.